std/range/package.d

```diff
@@ -2553,6 +2557,124 @@ auto takeNone(R)(R range)
 }
 
 /++
+ + Return a _range advanced to within $(D n) elements of the end of
+ + $(D _range).
+ +
+ + Intended as the _range equivalent of the Unix
+ + $(WEB en.wikipedia.org/wiki/Tail_%28Unix%29, _tail) utility. When the length
+ + of $(D _range) is less than or equal to $(D n), $(D _range) is returned
+ + as-is.
+ +
+ + Completes in $(BIGOH 1) steps for ranges that support slicing and have
+ + length. Completes in $(BIGOH _range.length) time for all other ranges.
+ +/
```

Comment:
I thought about `Params` and `Returns` but I couldn't think of a way of phrasing them without them being utter noise. The whole thing would have to be rewritten around those sections and I'm not sure the documentation would be better off for it.